src/azure-cli/azure/cli/command_modules/container/_params.py

```diff
@@ -79,6 +79,13 @@ def load_arguments(self, _):
 
     with self.argument_context('container create', arg_group='Confidential Container Group') as c:
         c.argument('cce_policy', help="The CCE policy for the confidential container group")
+        c.argument('allow_privilege_escalation', options_list=['--allow-escalation'], help="Allows whether a process can gain more privileges than its parent process.", action='store_true')
+        c.argument('privileged', help='The flag to determine if the contianer permissions is elevated to Privileged', action='store_true')
+        c.argument('run_as_user', help="Sets the User GID for the container")
+        c.argument('run_as_group', help="Sets the User UID for the container")
+        c.argument('seccomp_profile', help="A base64 encoded string containing the contents of the JSON in the seccomp profile")
+        c.argument('add_capabilities', type=str, nargs='+', help="A List of security context capabilities to be added")
+        c.argument('drop_capabilities', type=str, nargs='+', help="A List of security context capabilities to be dropped")
```

Comment:
`type=str` is the default setting, please do not explicitly repeat the declaration
```suggestion
        c.argument('add_capabilities', nargs='+', help="A List of security context capabilities to be added")
        c.argument('drop_capabilities', nargs='+', help="A List of security context capabilities to be dropped")
```

Reply:
> @fnuarnav I cannot reproduce your issue locally. Could you try using `az login` to login first, and then use `az account show` to check the subscription being used and then execute `azdev test test_tag_update_by_patch --live` to re-record the yaml file?

I have tried logging in and running the test but still gives the same error.

Reply:
@fnuarnav May I ask if the subscription output after executing `az login` meets your expectations?

Reply:
Yes, after `az login` the subscription output is what I would expect. I was able to run the tests under the container command_module with this subscription.